### ISSUE TITLE
Metis: browser-based download directory includes corrupt files

### ISSUE THREAD

**Issue by coleshaw** [mountetna/monoetna#882]
Reported by a user, that using the "download directory" button in the Metis UI on a large data set results in corrupt files present in the Zip. TB-size directory, so small tests may not replicate the issue.

**Comment by graft** [comment]
Is the directory known that we might attempt to replicate?

**Comment by coleshaw** [comment]
Hm...okay, so I would **try** to replicate the issue, except I don't have 1TB+ of space to do it with :sweat_smile: Anyone have a spare external drive with some free space?

**Comment by graft** [comment]
Hmm, so some file systems can't even make a file this big... Notably FAT, so if they were saving to an external FAT-formatted disk this might not work.

**Comment by coleshaw** [comment]
Hm, interesting, good point! Let me follow-up and see.

**Comment by coleshaw** [comment]
Yes, I believe it would be the main aip1 data bucket / folder or pilot data, though it sounded like they were trying to get all the raw project data, so my guess is not pilot data.
 
-----Original Message-----
